Sources/armory/logicnode/ClearParentNode.hx

```diff
@@ -19,9 +19,13 @@ class ClearParentNode extends LogicNode {
 
 		#if arm_physics
 		var rigidBody = object.getTrait(RigidBody);
-		if (rigidBody != null) rigidBody.syncTransform();
+
+		if (rigidBody != null) {
+		rigidBody.animated = false; // Workaround
```

Comment:
@knowledgenude , Thank you for the fixes,
I do not fully understand the use of this line here. What is its purpose? 
Setting `rigidBody.animated = false` only affects the trait and not the actual bullet rigid body.

Reply:
@QuantumCoderQC , this is not a correct fix, but it is the only way i found to get Set Parent working with rigid bodies. Note that i Set the animated setting trough Set Parent and in Clear Parent i have to disable animated or the rigid body will be not "active".